File: php/Grace.php
#!/usr/bin/php
<?PHP
/*
	Comment
*/

define('C', '#!/usr/bin/php%c<?PHP%c/%c%c	Comment%c%c/%c%cdefine(%cC%c, %c%s%c);%cdefine(%cA%c, %c10%c);%cdefine(%cB%c, %c34%c);%cfprintf(fopen(%cGrace_kid.php%c, %cw+%c), C, A, A, 42, A, A, 42, A, A, 39, 39, 39, C, 39, A, 39, 39, 39, 39, A, 39, 39, 39, 39, A, B, B, B, B, A);%c?>');
define('A', '10');
define('B', '34');
fprintf(fopen("Grace_kid.php", "w+"), C, A, A, 42, A, A, 42, A, A, 39, 39, 39, C, 39, A, 39, 39, 39, 39, A, 39, 39, 39, 39, A, B, B, B, B, A);
?>

File: Grace.c
#include <stdio.h>

/*
	Comment
*/

#define C "#include <stdio.h>%1$c%1$c/%2$c%1$c	Comment%1$c%2$c/%1$c%1$c#define C %3$c%4$s%3$c%1$c#define PRINT C, 10, 42, 34, C%1$c#define F int main(void){fprintf(fopen(%3$cGrace_kid.c%3$c, %3$cw+%3$c), PRINT);}%1$cF"
#define PRINT C, 10, 42, 34, C
#define F int main(void){fprintf(fopen("Grace_kid.c", "w+"), PRINT);}
F